packages/ia-icons/README.md

```diff
@@ -0,0 +1,31 @@
+# ia-icons: SVG icon library for use in archive.org components
+
+## Usage
+
+```js
+import { IconClose } from './src/ia-icons';
+
+customElements.define('icon-close', IconClose);
+```
+
+```html
+<icon-close></icon-close>
```

Comment:
Make sure to update the readme for the new syntax.

Reply:
:t_hanks: